docs/architecture/adr-005-dual-login-and-publizon-adapter.md

```diff
@@ -0,0 +1,67 @@
+# Dual Login and Publizon Adapter
+
+## Context
+
+Our application supports two different authentication mechanisms for users:
+
+- **Adgangsplatformen** — used by library patrons.
+- **Unilogin** — a login used by students, particularly children in schools.
+
+Depending on which method is used, the session is established differently and includes a session token that is used to authorize requests to external services.
+
+A key integration in our system is the **Publizon Adapter** ([see documentation here](https://library-api.qa.pubhub.dk/index.html)), a service that acts as a middle layer between us and the **Publizon Pubhub** (a SOAP-based backend-to-backend system). [See documentation here.](https://libraryservices.pubhub.dk/v2_7/) This adapter provides endpoints we use to achieve various things, such as:
```

Comment:
In our research we found out that the Publizon adapter actually is a very thin layer proxying request further to a Publizon API. So it is not correct that you write that it is a SOAP-based backend-to-backend system

Reply:
I think maybe we mean the same thing here. But if it's confusing to understand, I can reformulate it.